src/create-release.js

```diff
@@ -70,10 +70,14 @@ releaseNotes(ghRepo, argv)
     .then(notes => {
         log.info('\nRelease Notes:\n', notes);
         if (!argv['dry-run']) {
+            let tag = argv.tag;
+            if (tag.charAt(0) !== 'v') {
+                tag = `v${tag}`;
+            }
```

Comment:
I don't ❤️ this as it locks us into a specific tag format, but I am okay with it since it's the easier place to make a change and it's really only used in our own repos that use this tag format.